src/Logger/ProfilerLogger.php

```diff
@@ -77,14 +80,22 @@ public function startQuery($sql, ?array $params = null, ?array $types = null): v
 	 */
 	private function expandListParameters(string $query, array $params, array $types): array
 	{
-		if (class_exists(SQLParserUtils::class)) { // DBAL 2.x compatibility
+		if ($this->olderDbalVersion === null) {
```

Comment:
Why is here variable?

Reply:
Because expandListParameters is called on every query, and class_exists is countless times slower than checking boolean variable, so class_exists is called only once. For instance, if there were 1000 queries or more, in some console cron, it would slowed down the app for a few miliseconds, just because of that class_exists in every query.